app/src/main/java/mozilla/lockbox/support/DataStoreSupport.kt

```diff
@@ -42,25 +45,70 @@ class FixedDataStoreSupport(values: List<ServerPassword>? = null) : DataStoreSup
 
 /**
  * Creates a test ServerPassword item
+ * Some functionality inspired by FxA 'upload_fake_passwords.py'
+ * https://gist.github.com/rfk/916d9ca684f862b1c1030c685a5a4d19
  */
 internal fun createDummyItem(idx: Int): ServerPassword {
-    val rng = Random()
-    val pos = idx + 1
-    val pwd = "AAbbcc112233!"
-    val host = "https://$pos.example.com"
+    val random = Random()
+    val pwd = createRandomPassword()
+    val host = createHostname()
+    val user = createUserId()
     val created = Date().time
     val used = Date(created - 86400000).time
     val changed = Date(used - 86400000).time
 
     return ServerPassword(
-            id = "0000$idx",
-            hostname = host,
-            username = "someone #$pos",
-            password = pwd,
-            formSubmitURL = host,
-            timeCreated = created,
-            timeLastUsed = used,
-            timePasswordChanged = changed,
-            timesUsed = rng.nextInt(100) + 1
+        id = "0000$idx",
```

Comment:
this ought to be `UUID.randomUUID().toString()` ... the original implementer (me) was too lazy (-:

Reply:
I'll add the UUID impl 👍